src/Text/Pandoc/Writers/Custom.hs

```diff
@@ -25,31 +26,33 @@ import qualified Text.Pandoc.Lua.Writer.Classic as Classic
 -- | Convert Pandoc to custom markup.
 writeCustom :: (PandocMonad m, MonadIO m)
             => FilePath -> WriterOptions -> Pandoc -> m Text
-writeCustom luaFile opts doc = either throw pure <=< runLua $ do
-  setGlobals [ PANDOC_DOCUMENT doc
-             , PANDOC_SCRIPT_FILE luaFile
-             , PANDOC_WRITER_OPTIONS opts
-             ]
-  dofileTrace luaFile >>= \case
-    OK -> pure ()
-    _  -> throwErrorAsException
-  -- Most classic writers contain code that throws an error if a global
-  -- is not present. This would break our check for the existence of a
-  -- "Writer" function. We resort to raw access for that reason, but
-  -- could also catch the error instead.
-  let rawgetglobal x = do
-        pushglobaltable
-        pushName x
-        rawget (nth 2) <* remove (nth 2) -- remove global table
+writeCustom luaFile opts doc = do
+  luaFile' <- fromMaybe luaFile <$> findFileWithDataFallback "readers" luaFile
```

Comment:
should this be `"writers"` instead of `"readers"`?

Reply:
Whoops. Careless copy-pasting.